src/plugins/plugin.text_selection.js

```diff
@@ -273,7 +274,20 @@ export class BookreaderWithTextSelection extends BookReader {
       // contains a reference to it.
       this.options.plugins.textSelection = options;
       this.textSelectionPlugin.init();
+
+      // Track how often selection is used
+      const sso = new SelectionStartedObserver('.textSelectionSVG', () => {
+        // Don't assume the order of the plugins ; the analytics plugin could
+        // have been added later. But at this point we should no for certain.
+        if (!this.archiveAnalyticsSendEvent) {
+          sso.detach();
+        } else {
+          this.archiveAnalyticsSendEvent?.('BookReader', 'SelectStart');
+        }
+      });
+      sso.attach();
```

Comment:
do you need to pin this somewhere to detach later?

Reply:
I have! I detach it weirdly just above :P